src/DurableTask.Netherite/StorageLayer/Faster/FaultInjector.cs

```diff
@@ -78,6 +83,14 @@ public void Dispose()
         readonly Dictionary<int, TaskCompletionSource<object>> startupWaiters = new Dictionary<int, TaskCompletionSource<object>>();
         readonly HashSet<BlobManager> startedPartitions = new HashSet<BlobManager>();
 
+        public void StartClient()
```

Comment:
I would use same name as for flag, so if the flag is called `FailClientStartup` lets call this `ClientStartup`

Reply:
good idea, applied this feedback.